src/modules/GroupChannelList/context/GroupChannelListProvider.tsx

```diff
@@ -87,27 +87,26 @@ const useGroupChannelListStore = () => {
   return useStore(GroupChannelListContext, state => state, initialState);
 };
 
-export const GroupChannelListManager: React.FC<GroupChannelListProviderProps> = (props: GroupChannelListProviderProps) => {
-  const {
-    className = '',
-    selectedChannelUrl,
+export const GroupChannelListManager: React.FC<GroupChannelListProviderProps> = ({
+  className = '',
+  selectedChannelUrl = '',
 
-    disableAutoSelect = false,
-    allowProfileEdit,
-    isTypingIndicatorEnabled,
-    isMessageReceiptStatusEnabled,
+  disableAutoSelect = false,
+  allowProfileEdit = false,
+  isTypingIndicatorEnabled = false,
+  isMessageReceiptStatusEnabled = false,
```

Comment:
```suggestion
  allowProfileEdit,
  isTypingIndicatorEnabled,
  isMessageReceiptStatusEnabled,
```
We'd better to keep the original status of this prop as-is since there might be a chance that some behavior changes due to this default value setting.

Reply:
Applied.